src/Pages/Login/SignUp.js

```diff
@@ -0,0 +1,321 @@
+                확인하고 전체 동의합니다. ※ 선택 항목에 대한 동의를 거부하더라도
+                회원가입에 영향을 미치지 않습니다.
+              </Explanation>
+              <SignUpSubtitile>
+                아모레퍼시픽 통합 멤버십 뷰티포인트 회원약관
+              </SignUpSubtitile>
+              <label>
+                <input type="checkbox" />
+                [필수]뷰티포인트 서비스 이용약관
+              </label>
+              <label>
+                <input type="checkbox" />
+                [필수]개인정보 이용 및 수집에 대한 동의
+              </label>
+              <label>
+                <input type="checkbox" />
+                [선택]개인정보 제3자 제공 동의
+              </label>
+              <label>
+                <input type="checkbox" />
+                [선택]국외 이전 동의
+              </label>
+              <SignUpSubtitile>
+                광고성 정보 수신 동의
+                <span>쇼핑 혜택, 이벤트 소식을 받아보세요</span>
+              </SignUpSubtitile>
+              <label>
+                <input type="checkbox" />
+                [선택]뷰티포인트 문자 수신 동의
+              </label>
+              <label>
+                <input type="checkbox" />
+                [선택]온라인 사이트 문자 수신 동의
+              </label>
+              <SignUpSubtitile>개인정보 처리 위탁에 대한 안내</SignUpSubtitile>
+              <Explanation>
+                ※ 아모레퍼시픽은 서비스 향상 및 원활한 전산 처리 등을 위하여
+                이용자의 개인정보 관리를 외부 전문업체에 위탁하고 있습니다.
+                아모레퍼시픽의 업무를 위탁받는 자 및 업무의 내용은 아모레퍼시픽
+                홈페이지
+                https://www.beautypoint.co.kr/footer/privacy/personal.html 에서
+                확인 하실 수 있습니다.
+              </Explanation>
+            </TermsList>
+          </ArticleCol>
+          <Link to="?">
+            <SignUpButton agree="true">본인인증 및 회원가입</SignUpButton>
+          </Link>
+        </section>
+      </SignUpMain>
+    </Signup>
+  );
+}
+
+const Signup = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100vw;
+  font-family: "MalgunGothic";
+`;
+
+const SignUpHeader = styled.div`
+  border: 1px solid #d2d2d2;
+  width: inherit;
+  height: 61px;
+  font-size: 16px;
+  font-weight: 700;
+  letter-spacing: 1px;
+  text-align: center;
+  color: #082b5e;
+  padding-top: 23px;
+  background-color: #f3f3f3;
+`;
+
+const SignUpMain = styled.main`
+  width: 460px;
+
+  h1 {
+    font-size: 30px;
+    margin-top: 40px;
+  }
+
+  h3 {
+    font-size: 15px;
+    margin-top: 10px;
+    font-weight: 400;
+    letter-spacing: 0.8px;
+  }
+
+  p {
+    margin-bottom: 15px;
+  }
+
+  .idAlert {
+    font-size: 14px;
+    font-weight: 700;
+  }
+
+  section {
+    margin-bottom: 70px;
+  }
+`;
+
+const Article = styled.article`
+  display: flex;
+  margin-bottom: 15px;
+
+  input[type="text"],
+  input[type="password"],
+  input[type="number"],
+  select {
+    width: 100%;
+    height: 54px;
+    font-size: 16px;
+    border: 1px solid #d2d2d2;
+    padding-left: 10px;
+    &::placeholder {
+      font-size: 16px;
+    }
+  }
+`;
+
+const ArticleCol = styled(Article.withComponent("article"))`
```

Comment:
```suggestion
const ArticleCol = styled(Article)`
```
다른 태그로 바꿔줄거 아니기 때문에 그냥 쓰시면 됩니다

Reply:
오 알겠습니다